src/components/Footer/index.js

```diff
@@ -3,23 +3,63 @@ import React from 'react';
 import { withRouter, Link } from 'react-router-dom';
 import { connect } from 'react-redux';
 import Fab from '@material-ui/core/Fab';
-import AddIcon from '@material-ui/icons/Add';
+import { AppBar } from '@material-ui/core';
 import useStyles from './style';
 import MetamaskGateway from '../MetamaskGateway';
 
 
+import Toolbar from '@material-ui/core/Toolbar';
+import IconButton from '@material-ui/core/IconButton';
+import MenuIcon from '@material-ui/icons/Menu';
+import AccountCircle from '@material-ui/icons/AccountCircle';
+import AddAPhoto from '@material-ui/icons/AddAPhoto';
+
+import Grid from '@material-ui/core/Grid';
+
+
+
 const Footer = ({ currentUser }) => {
   const classes = useStyles();
   const isLogedIn = currentUser.jwt || currentUser.djwt;
 
   return (
-    <>
+      <>
       {isLogedIn && (
-        <MetamaskGateway>
-          <Fab color="primary" aria-label="add" className={classes.fab} component={Link} to="report"><AddIcon /></Fab>
-        </MetamaskGateway>
-      )}
-    </>
+        <AppBar position="fixed" color="primary" className={classes.appBar}>
+          <Toolbar>
+            <Grid container text-align="center">
+              <Grid container item xs={12} spacing={0} justify="center">
```

Comment:
- [ ] text-align="center"
```suggestion
              <Grid container item xs={12} spacing={0} justify="center" text-align="center" >
```